module/Application/view/layout/layout.phtml

```diff
@@ -107,7 +107,12 @@
             </div>
         </div>
     </header>
-
+    <div id="security-only-notice" class="navbar-expand-lg">
+        <p class="container">
+            Laminas API Tools is considered feature-complete, and is in security-only maintenance mode. An alternative can be found
+            <a href="https://getlaminas.org/blog/2024-07-02-dotkernel-a-replacement-for-laminas-api-tools.html" target="_blank">here.</a>
+        </p>
```

Comment:
My suggestion:

```suggestion
            Laminas API Tools is considered <a href="https://getlaminas.org/blog/2024-07-02-dotkernel-a-replacement-for-laminas-api-tools.html" target="_blank"><strong>feature-complete</strong></a>, and is in <strong>security-only maintenance mode</strong>. Check out the alternative:
            <a href="https://www.dotkernel.org" target="_blank">Dotkernel API</a>
        </p>
```

The text decoration for the links should be set to `underline`, otherwise it is difficult to identify the links.

Reply:
![image](https://github.com/user-attachments/assets/5e6a9694-22f1-4e6c-a2b1-834044665702)

updated